lib/open_project/static/links.rb

```diff
@@ -67,7 +67,7 @@ def links
               label: 'homescreen.links.shortcuts'
             },
             boards: {
-              href: 'https://community.openproject.com/projects/openproject/boards',
+              href: 'https://community.openproject.com/projects/openproject/forums',
               label: 'homescreen.links.boards'
```

Comment:
This should be `forums`. And also on L69.

Reply:
Thanks!